src/BotService/Infrastructure/Core/Bot.cs

```diff
@@ -131,7 +131,9 @@ public async Task RemoveBotAsync(string callGraphId)
         {
             try
             {
-                await GetHandlerOrThrow(callGraphId).Call.DeleteAsync().ConfigureAwait(false);
+                var callHandler = GetHandlerOrThrow(callGraphId);
+                callHandler.StopAllExtractions();
```

Comment:
**suggestion**: We also need to stop the injection (if running), not only the extractions.

Reply:
@sw-dcherubini Injections are already being stopped in the dispose of the callhandler, but we could not use the same approach for extractions because if we try to stop extractions in the dispose, after the call is removed, the unsubscribe of the videoSocket in the mediaExtractor will throw an exception